src/ledger/balances.ts

```diff
@@ -1,30 +1,29 @@
 import * as utils from './utils'
 import {validate, ensureClassicAddress} from '../common'
-import {Connection} from '../client'
 import {GetTrustlinesOptions} from './trustlines'
 import {FormattedTrustline} from '../common/types/objects/trustlines'
 import {Client} from '..'
 
 export type Balance = {
   value: string
   currency: string
-  counterparty?: string
+  issuer?: string
 }
 
 export type GetBalances = Array<Balance>
 
 function getTrustlineBalanceAmount(trustline: FormattedTrustline): Balance {
   return {
     currency: trustline.specification.currency,
-    counterparty: trustline.specification.counterparty,
+    issuer: trustline.specification.issuer,
     value: trustline.state.balance
   }
 }
```

Comment:
What is the purpose of this? It doesn't do anything

Reply:
Deleted